doc/source/building_images/build_container_image.rst

```diff
@@ -0,0 +1,103 @@
+.. _building_container_build:
+
+Build a Container Image
+=======================
+
+.. sidebar:: Abstract
+
+   This page explains how to build a Container Image. It contains
+
+   * basic configuration explanation
+   * how to build a Container Image
+   * how to run it with a Container Runtime
+
+{kiwi} is capable of building native Container Images from scratch and
+derived ones. {kiwi} Container images are considered to be native since the
+{kiwi} tarball image is ready to be loaded into a Container Runtime like
+Podman, Docker or Containerd, including common container configurations.
+
+The Container configuration metadata is provided to {kiwi} as part of the
+:ref:`XML description file <description_components>` using the
+``<containerconfig>`` tag. The following configuration metadata can be
+specified:
+
+`containerconfig` attributes:
+
+* ``name``: Specifies the repository name of the Container Image.
+* ``tag``: Sets the tag of the Container Image.
+* ``maintainer``: Specifies the author field of the container, this is
+  equivalent to the `MAINTAINER` directive in a :file:`Dockerfile`.
+* ``user``: Sets the user name or user id (UID) to be used when
+  running `entrypoint` and `subcommand`. Equivalent of the `USER`
+  directive of a :file:`Dockerfile`.
+* ``workingdir``: Sets the working directory to be used when running
+  `cmd` and `entrypoint`. Equivalent of the `WORKDIR` directive in a
+  :file:`Dockerfile`.
+
+`containerconfig` child tags:
+
+* ``subcommand``: Provides the default execution parameters of the
+  container. Equivalent of the `CMD` directive in a :file:`Dockerfile`.
+* ``labels``: Adds custom metadata to an image using key-value pairs.
+  Equivalent to one or more `LABEL` directives in a :file:`Dockerfile`.
+* ``expose``: Define which ports can be exposed to the outside when
+  running this container image.  Equivalent to one or more `EXPOSE`
+  directives in a :file:`Dockerfile`.
+* ``environment``: Sets environment variables using key-value pairs.
+  Equivalent to one or more the `ENV` directives of a
+  :file:`Dockerfile`.
+* ``entrypoint``: Sets the binary via which all commands inside the
```

Comment:
via which -> from which ?

Reply:
Not really, because whatever is written in `entrypoint` calls your supplied parameters, so I don't think that "from which" is a good fit here.

Reply:
ok